HLTrigger/Muon/plugins/HLTScoutingTrackProducer.h

```diff
@@ -0,0 +1,62 @@
+#ifndef HLTScoutingTrackProducer_h
+#define HLTScoutingTrackProducer_h
+
```

Comment:
It is best to remove this file and put its content directly in the corresponding .cc file.